src/components/FormContainer/FormContainer.scss

```diff
@@ -58,3 +61,60 @@
   background: url('../../resources/images/Purple_triangle.png') 0 0 repeat;
   transform: rotate(30deg);
 }
+
+@include xlg {
+  .form-page-container {
+    &.register {
+      width: 80vw;
+    }
+  }
+}
+
+@include lg {
+  .form-page-container {
+    &.register {
+      width: 90vw;
+    }
+  }
+}
+
+@include md {
+  .form-page-container {
+    &.register,
+    &.login {
+      width: 50vw;
+    }
+  }
+
+  .form-container {
+    border-top-left-radius: 10px;
+    border-bottom-left-radius: 10px;
+  }
+
+  .image-container {
+    display: none;
+  }
+
+  .blue-triangle1 {
+    display: none;
+  }
+}
+
```

Comment:
Please, check how to fix the container of Login and Register forms for a mobile landscape view.